exporters/metric/cortex/cortex.go

```diff
@@ -69,6 +93,231 @@ func InstallNewPipeline(config Config, options ...push.Option) (*push.Controller
 	return pusher, nil
 }
 
+// ConvertToTimeSeries converts a CheckpointSet to a slice of TimeSeries pointers
+func (e *Exporter) ConvertToTimeSeries(checkpointSet export.CheckpointSet) ([]*prompb.TimeSeries, error) {
+	var aggError error
+	var timeSeries []*prompb.TimeSeries
+
+	// Iterate over each record in the checkpoint set and convert to TimeSeries
+	aggError = checkpointSet.ForEach(e, func(record metric.Record) error {
+		// Convert based on aggregation type
+		agg := record.Aggregation()
+
+		// Check if aggregation has Sum value
+		if sum, ok := agg.(aggregation.Sum); ok {
+			ts, err := convertFromSum(record, sum)
+			if err != nil {
+				return err
+			}
+
+			timeSeries = append(timeSeries, ts)
+		}
+
+		// Check if aggregation has MinMaxSumCount value
+		if mmsc, ok := agg.(aggregation.MinMaxSumCount); ok {
```

Comment:
I don't like this variable name, mmsc

Reply:
mmsc is an abbreviation of MinMaxSumCount and is used consistently in the other metrics exporters. Same for the others (lv = lastValue, dist = distribution, etc). 

I'll expand them out to make them more clear.